tests/test_handle_html_content.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+from unittest.mock import Mock
+import asyncio
+import argparse
+import aiohttp
+import shutil
+import os
+import yarl
+from bs4 import BeautifulSoup
+from snare import HttpRequestHandler
+
+
+class AsyncMock(Mock):  # custom function defined to mock asyncio coroutines
+
+    def __call__(self, *args, **kwargs):
+        sup = super(AsyncMock, self)
+
+        async def coro():
+            return sup.__call__(*args, **kwargs)
+        return coro()
+
+    def __await__(self):
+        return self().__await__()
+
+
+class TestHandleHtmlContent(unittest.TestCase):
+    def setUp(self):
+        self.meta = {}
+        run_args = argparse.ArgumentParser()
+        run_args.add_argument("--tanner")
+        run_args.add_argument("--page-dir")
+        run_args.add_argument("--no-dorks")
+        if not os.path.exists("/opt/snare/pages/test"):
```

Comment:
what happens if someone has directory target 'test"?

Reply:
If someone has directory target "test" this test will automatically delete it at the end of the test.
I think we should check the existence of the directory and leave it be if it already exists after the test. Or delete the folder if we created it in the test.

Reply:
Please check if it is okay now. :)